README.md

```diff
@@ -111,19 +111,25 @@ npm run test:headless:watch
 
 This repo uses the [@brightspace-ui/visual-diff utility](https://github.com/BrightspaceUI/visual-diff/) to compare current snapshots against a set of golden snapshots stored in source control.
 
+The golden snapshots in source control must be updated by Github Actions.  If your PR's code changes result in visual differences, a PR with the new goldens will be automatically opened for you against your branch.
+
+If you'd like to run the tests locally to help troubleshoot or develop new tests, you can use these commands:
+
 ```shell
+# Install dependencies locally
+npm i mocha -g
+npm i @brightspace-ui/visual-diff puppeteer --no-save
+
```

Comment:
I'll take a look at this whole README snippet when we port this over to the `npm init` repo.